mm2src/mm2_main/tests/mm2_tests/tendermint_tests.rs

```diff
@@ -740,6 +742,8 @@ mod swap {
     }
 
     #[test]
+    // runs "forever" for some reason
+    #[ignore]
```

Comment:
I wouldn't ignore the test if it failed in a timely manner, but at least in my environment, it ran indefinitely long. I will recheck this test and get back here shortly.

Reply:
I meant that these tests are failing because something has been changed on the iris nodes. It's not due to your environment or PR. They are also failing on other branches. I plan to investigate the change in the coming week.

Reply:
The key point of my comment was that they run *indefinitely long*. If the same happens on CI, we will have to wait for 90 minutes workflow timeout to get the actual result. So, I assumed it would be more optimal to ignore them. I wouldn't do it if they failed in a timely manner.

Reply:
> The key point of my comment was that they run indefinitely long.

This is strange as we have our own timeouts in those tests. I guess it's ok to disabling them in this case.